wtpython/__main__.py

```diff
@@ -29,34 +31,38 @@ def parse_arguments() -> tuple[dict, list]:
     )
 
     flags, args = parser.parse_known_args()
+
+    # Set sys.argv as the intended script would receive them
+    sys.argv = ['python', *args[1:]]
```

Comment:
`argv[0]` should be the name of the script I believe, not `python`. You should just be able to do `sys.argv = args`.

This does seem a bit hacky, so maybe let's just replace `argv` right before we run the script with `runpy`, and then restore the original value right after.

Reply:
it works either way, but you're right `sys.argv = args` is a bit cleaner.